assets/site.scss

```diff
@@ -3,261 +3,74 @@
 
 @import "colors.scss";
 
-.page-content {
-  margin-top: 35px;
-  overflow: hidden;
-}
 
-.sub-heading {
-  font-weight: 100;
-  color: #353430;
-  line-height: 1.16666666666667;
-  line-height: 1.75rem; /* 28px */
-  margin-top: 60px;
-  margin-bottom:50px;
-  padding-bottom: 20px;
-  border-bottom: 1px dashed #BBB;
-  font: "proxima-nova-soft","Helvetica Neue","Microsoft Sans Serif",sans-serif;
-}
+
+	
+
 
 /* *** Color Palette **** */
 .palette {
-  padding: 10px;
+	padding: 10px;
 }
 
 .standard, .backgrounds {
-  margin-top: 0.5em;
-  border: solid $black 0.8px;
-  border-radius: 3px;
-  display: inline-block;
-  width: 220px;
-  box-sizing: border-box;
-  text-align: center;
-  padding: 15px;
-  box-shadow:0px 1px 0px 0px rgba(0,0,0,0.1),0px 2px 1px 0px rgba(0,0,0,0.12);
+	margin-top: 0.5em;
+	border: solid $black 0.8px;
+	border-radius: 3px;
+	display: inline-block;
+	width: 220px;
+	box-sizing: border-box;
+	text-align: center;
+	padding: 15px;
+	box-shadow:0px 1px 0px 0px rgba(0,0,0,0.1),0px 2px 1px 0px rgba(0,0,0,0.12);
```

Comment:
These settings to look 6-spaces.

Team, what rule do we want for CSS file, 2 or 4 spaces?